lite-pm/pages/components/AppPage.js

```diff
@@ -64,42 +282,25 @@ class AppPage extends Component {
 
   addTeamMember() {
     this.toggleAddMemberModal();
-    let members = this.state.teamMembers;
-    members.push(
-      <TeamMember
-        key={uuid()}
-        name={this.state.memberName}
-        email={this.state.memberEmail}
-        git={this.state.memberGit}
-        phone={this.state.memberPhone}
-      ></TeamMember>
-    );
-
-    let timelines = this.state.memberTimelines;
-    timelines.push(
-      <MemberTimeline
-        key={uuid()}
-        name={this.state.memberName}
-      ></MemberTimeline>
-    );
 
     //This creates members into the database
     //const queryString = window.location.search;
     //const urlParams = new URLSearchParams(queryString)
     //const projectId = urlParams.get('projectId');
     //Used for testing need to remove after for production
     const projectId = this.state.project.projectId;
+    console.log(projectId);
     axios.post(`${baseUrl}/api/members/create`, {
       projectId: projectId,
       name: this.state.memberName,
       email: this.state.memberEmail,
       github: this.state.memberGit,
       phone: this.state.memberPhone,
-    });
+    }).then(()=>{this.getProjectDetails()});
```

Comment:
Just a note so we remember to use the return from axios to populate the state instead of making a second call